ts-tests/scripts/clean-up.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+killall litentry-token-server
+killall polkadot
+killall litentry-collator
```

Comment:
If the tests failed, I think the pipeline will be broken and `clean-up.sh` will never be executed. @h4n0

Redirecting `stdout` to `/tmp/xxxx` can help us debug. And the file size won't increase dramatically since the processes will be killed when the `ci-test` is done.

Reply:
I don't think so, as there's no `set -e`, but even if does, the binaries won't be killed if clean-up.sh is not run.

I changed it so that:

- it kills the binaries in all cases, regardless of failed or ok tests.
- remove the logs if tests were successful
- keep the logs and print the log-directory if tests failed => for the following analysis

but I agree that killall is not the best solution.
Maybe I could have kept the PID of each binary process and only kills them later. @chenzongxiong  @h4n0

Reply:
Yeah. I agree. I think we can create an issue for this problem

Reply:
This is a good point. As my understanding `set -o pipefail` won't terminate the script but just return error after the script runs to the end. 
Best way to implement all these binaries is to spawn them in TS code and they will be killed automatically when the main thread is killed. However due to some limitations we did not achieve this.

Reply:
@Kailai-Wang Can you create an issue for me ? I'd like to use `pidfile` instead of `killall`